2018-06-01 09:36:20,813 INFO: Microblog startup [in c:\Pythonfiles\microblog\app\__init__.py:43]
2018-06-01 09:50:14,521 INFO: Microblog startup [in c:\Pythonfiles\microblog\app\__init__.py:43]
2018-06-01 09:50:29,479 INFO: Microblog startup [in c:\Pythonfiles\microblog\app\__init__.py:43]
2018-06-01 09:50:37,976 INFO: Microblog startup [in c:\Pythonfiles\microblog\app\__init__.py:43]
2018-06-01 09:51:00,566 INFO: Microblog startup [in c:\Pythonfiles\microblog\app\__init__.py:43]
2018-06-01 09:51:14,027 INFO: Microblog startup [in c:\Pythonfiles\microblog\app\__init__.py:43]
2018-06-01 12:00:45,347 INFO: Microblog startup [in c:\Pythonfiles\microblog\app\__init__.py:43]
2018-06-01 12:02:02,418 INFO: Microblog startup [in c:\Pythonfiles\microblog\app\__init__.py:43]
2018-06-01 12:02:37,984 INFO: Microblog startup [in c:\Pythonfiles\microblog\app\__init__.py:43]
2018-06-01 12:02:42,569 INFO: Microblog startup [in c:\Pythonfiles\microblog\app\__init__.py:43]
2018-06-01 12:03:02,730 INFO: Microblog startup [in c:\Pythonfiles\microblog\app\__init__.py:43]
2018-06-01 12:03:07,285 INFO: Microblog startup [in c:\Pythonfiles\microblog\app\__init__.py:43]
2018-06-01 12:03:21,827 INFO: Microblog startup [in c:\Pythonfiles\microblog\app\__init__.py:43]
2018-06-01 12:03:38,024 INFO: Microblog startup [in c:\Pythonfiles\microblog\app\__init__.py:43]
2018-06-01 12:03:56,786 INFO: Microblog startup [in c:\Pythonfiles\microblog\app\__init__.py:43]
2018-06-01 12:04:34,900 INFO: Microblog startup [in c:\Pythonfiles\microblog\app\__init__.py:43]
2018-06-01 12:06:34,523 INFO: Microblog startup [in C:\Pythonfiles\microblog\app\__init__.py:43]
2018-06-01 12:07:38,921 INFO: Microblog startup [in C:\Pythonfiles\microblog\app\__init__.py:43]
2018-06-01 12:07:46,290 INFO: Microblog startup [in C:\Pythonfiles\microblog\app\__init__.py:43]
2018-06-01 12:07:58,218 INFO: Microblog startup [in C:\Pythonfiles\microblog\app\__init__.py:43]
2018-06-01 12:08:00,707 INFO: Microblog startup [in C:\Pythonfiles\microblog\app\__init__.py:43]
2018-06-01 12:10:50,053 INFO: Microblog startup [in C:\Pythonfiles\microblog\app\__init__.py:43]
2018-06-01 12:11:27,850 INFO: Microblog startup [in C:\Pythonfiles\microblog\app\__init__.py:43]
2018-06-01 12:11:39,608 INFO: Microblog startup [in C:\Pythonfiles\microblog\app\__init__.py:43]
2018-06-01 12:11:45,469 INFO: Microblog startup [in C:\Pythonfiles\microblog\app\__init__.py:43]
2018-06-01 12:11:50,110 INFO: Microblog startup [in C:\Pythonfiles\microblog\app\__init__.py:43]
2018-06-01 12:11:59,031 INFO: Microblog startup [in C:\Pythonfiles\microblog\app\__init__.py:43]
2018-06-01 12:12:45,086 INFO: Microblog startup [in C:\Pythonfiles\microblog\app\__init__.py:43]
2018-06-01 12:14:01,587 INFO: Microblog startup [in C:\Pythonfiles\microblog\app\__init__.py:43]
2018-06-01 12:15:27,053 INFO: Microblog startup [in C:\Pythonfiles\microblog\app\__init__.py:43]
2018-06-01 12:15:54,674 INFO: Microblog startup [in C:\Pythonfiles\microblog\app\__init__.py:43]
2018-06-01 12:18:07,170 INFO: Microblog startup [in c:\Pythonfiles\microblog\app\__init__.py:43]
2018-06-01 12:27:45,343 INFO: Microblog startup [in c:\Pythonfiles\microblog\app\__init__.py:43]
2018-06-01 12:27:50,760 ERROR: Exception on /user/Ward2 [GET] [in c:\pythonfiles\microblog\venv\lib\site-packages\flask\app.py:1761]
Traceback (most recent call last):
  File "c:\pythonfiles\microblog\venv\lib\site-packages\flask\app.py", line 2292, in wsgi_app
    response = self.full_dispatch_request()
  File "c:\pythonfiles\microblog\venv\lib\site-packages\flask\app.py", line 1815, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "c:\pythonfiles\microblog\venv\lib\site-packages\flask\app.py", line 1718, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "c:\pythonfiles\microblog\venv\lib\site-packages\flask\_compat.py", line 35, in reraise
    raise value
  File "c:\pythonfiles\microblog\venv\lib\site-packages\flask\app.py", line 1813, in full_dispatch_request
    rv = self.dispatch_request()
  File "c:\pythonfiles\microblog\venv\lib\site-packages\flask\app.py", line 1799, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "c:\pythonfiles\microblog\venv\lib\site-packages\flask_login\utils.py", line 261, in decorated_view
    return func(*args, **kwargs)
  File "c:\Pythonfiles\microblog\app\routes.py", line 74, in user
    return render_template('user.html', user=user, posts=posts)
  File "c:\pythonfiles\microblog\venv\lib\site-packages\flask\templating.py", line 134, in render_template
    return _render(ctx.app.jinja_env.get_or_select_template(template_name_or_list),
  File "c:\pythonfiles\microblog\venv\lib\site-packages\jinja2\environment.py", line 869, in get_or_select_template
    return self.get_template(template_name_or_list, parent, globals)
  File "c:\pythonfiles\microblog\venv\lib\site-packages\jinja2\environment.py", line 830, in get_template
    return self._load_template(name, self.make_globals(globals))
  File "c:\pythonfiles\microblog\venv\lib\site-packages\jinja2\environment.py", line 804, in _load_template
    template = self.loader.load(self, name, globals)
  File "c:\pythonfiles\microblog\venv\lib\site-packages\jinja2\loaders.py", line 125, in load
    code = environment.compile(source, name, filename)
  File "c:\pythonfiles\microblog\venv\lib\site-packages\jinja2\environment.py", line 591, in compile
    self.handle_exception(exc_info, source_hint=source_hint)
  File "c:\pythonfiles\microblog\venv\lib\site-packages\jinja2\environment.py", line 780, in handle_exception
    reraise(exc_type, exc_value, tb)
  File "c:\pythonfiles\microblog\venv\lib\site-packages\jinja2\_compat.py", line 37, in reraise
    raise value.with_traceback(tb)
  File "c:\Pythonfiles\microblog\app\templates\user.html", line 28, in template
    {% endblock %}
  File "c:\pythonfiles\microblog\venv\lib\site-packages\jinja2\environment.py", line 497, in _parse
    return Parser(self, source, name, encode_filename(filename)).parse()
  File "c:\pythonfiles\microblog\venv\lib\site-packages\jinja2\parser.py", line 901, in parse
    result = nodes.Template(self.subparse(), lineno=1)
  File "c:\pythonfiles\microblog\venv\lib\site-packages\jinja2\parser.py", line 883, in subparse
    rv = self.parse_statement()
  File "c:\pythonfiles\microblog\venv\lib\site-packages\jinja2\parser.py", line 130, in parse_statement
    return getattr(self, 'parse_' + self.stream.current.value)()
  File "c:\pythonfiles\microblog\venv\lib\site-packages\jinja2\parser.py", line 268, in parse_block
    node.body = self.parse_statements(('name:endblock',), drop_needle=True)
  File "c:\pythonfiles\microblog\venv\lib\site-packages\jinja2\parser.py", line 165, in parse_statements
    result = self.subparse(end_tokens)
  File "c:\pythonfiles\microblog\venv\lib\site-packages\jinja2\parser.py", line 883, in subparse
    rv = self.parse_statement()
  File "c:\pythonfiles\microblog\venv\lib\site-packages\jinja2\parser.py", line 130, in parse_statement
    return getattr(self, 'parse_' + self.stream.current.value)()
  File "c:\pythonfiles\microblog\venv\lib\site-packages\jinja2\parser.py", line 213, in parse_if
    'name:endif'))
  File "c:\pythonfiles\microblog\venv\lib\site-packages\jinja2\parser.py", line 165, in parse_statements
    result = self.subparse(end_tokens)
  File "c:\pythonfiles\microblog\venv\lib\site-packages\jinja2\parser.py", line 883, in subparse
    rv = self.parse_statement()
  File "c:\pythonfiles\microblog\venv\lib\site-packages\jinja2\parser.py", line 144, in parse_statement
    self.fail_unknown_tag(token.value, token.lineno)
  File "c:\pythonfiles\microblog\venv\lib\site-packages\jinja2\parser.py", line 97, in fail_unknown_tag
    return self._fail_ut_eof(name, self._end_token_stack, lineno)
  File "c:\pythonfiles\microblog\venv\lib\site-packages\jinja2\parser.py", line 90, in _fail_ut_eof
    self.fail(' '.join(message), lineno)
  File "c:\pythonfiles\microblog\venv\lib\site-packages\jinja2\parser.py", line 59, in fail
    raise exc(msg, lineno, self.name, self.filename)
jinja2.exceptions.TemplateSyntaxError: Encountered unknown tag 'endblock'. You probably made a nesting mistake. Jinja is expecting this tag, but currently looking for 'elif' or 'else' or 'endif'. The innermost block that needs to be closed is 'if'.
2018-06-01 12:29:43,844 INFO: Microblog startup [in c:\Pythonfiles\microblog\app\__init__.py:43]
2018-06-01 12:30:41,911 INFO: Microblog startup [in c:\Pythonfiles\microblog\app\__init__.py:43]
2018-06-01 23:38:25,791 INFO: Microblog startup [in c:\Pythonfiles\microblog\app\__init__.py:43]
2018-06-01 23:41:56,365 INFO: Microblog startup [in c:\Pythonfiles\microblog\app\__init__.py:45]
2018-06-03 12:23:37,825 INFO: Microblog startup [in c:\Pythonfiles\microblog\app\__init__.py:45]
2018-06-03 12:24:38,175 INFO: Microblog startup [in c:\Pythonfiles\microblog\app\__init__.py:45]
2018-06-03 12:27:30,582 INFO: Microblog startup [in c:\Pythonfiles\microblog\app\__init__.py:45]
2018-06-03 12:28:03,121 INFO: Microblog startup [in c:\Pythonfiles\microblog\app\__init__.py:45]
2018-06-03 14:38:05,531 INFO: Microblog startup [in C:\Pythonfiles\microblog\app\__init__.py:45]
2018-06-03 14:39:54,272 INFO: Microblog startup [in C:\Pythonfiles\microblog\app\__init__.py:45]
2018-06-03 14:49:40,596 INFO: Microblog startup [in C:\Pythonfiles\microblog\app\__init__.py:45]
2018-06-03 17:38:38,092 INFO: Microblog startup [in c:\Pythonfiles\microblog\app\__init__.py:45]
